modules/ROOT/pages/configuration-schema-db-security.adoc

```diff
@@ -81,16 +109,99 @@ Content-Length: 67
 
 =====
 
-<.> Add role `newrole` to the `travel25` database
-<.> Use Basic Auth to authenticate to an existing RBAC user
+<.> Add role `newrole` to the `travel25` database and the `scopename` scope and `collectionname` collection
+<.> Note we are using Basic Auth to authenticate to an existing RBAC user
+
+====
+
+=== Example using Default collections
 
 ====
 
+[{tabs}]
+=====
+
+Curl::
++
+--
+
+[source, bash]
+----
+
+curl --location --request PUT 'http://127.0.0.1:4985/travel25/_role/newrole' \ // <.>
+--header 'Authorization: Basic c3luY19nYXRld2F5OnBhc3N3b3Jk' \ // <.>
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "name": "newrole",
+    "collection_access": {
+      "_default": {
+          "_default": {
+               "admin_channels": ["newrolechannel"]
+          }
+      }
+   }
+}'
+
+----
+
+--
+
+HTTP::
++
+--
+
+[source, bash]
+----
+
+PUT /travel25/_role/newrole HTTP/1.1 // <.>
+Host: 127.0.0.1:4985
+Authorization: Basic c3luY19nYXRld2F5OnBhc3N3b3Jk // <.>
+Content-Type: application/json
+Content-Length: 132
+
+{
+    "name": "newrole",
+    "collection_access": {
+      "_default": {
+          "_default": {
+               "admin_channels": ["newrolechannel"]
+          }
+      }
+   }
+}
+
+----
 
+--
 
-include::partial$static_restapi/admin/index.adoc[tags=put_db-_user-name,leveloffset=+1]
+=====
 
-=== Example
+<.> Add role `newrole` to the `travel25` database using the `_default` scope and collection
+<.> Note we are using Basic Auth to authenticate to an existing RBAC user
+
+====
+
+For complete role management operations, see: {rest-api-admin-role-post--xref}[Role management API Reference]
+
+
+[#lbl-upsert-user]
+== Create or Update a User
+
+----
+PUT /{db}/_user/{username} // <.>
+----
+
+<.> where `{db}` is the Sync Gateway database and `{username}` is the name for the new user
+
+include::partial$static_restapi/admin/index.adoc[tags=put_db-_user-name-description,leveloffset=-3]
+
+=== Parameters
+include::partial$static_restapi/admin/index.adoc[tags=put_db-_user-name-parameters,leveloffset=-3]
+
+=== Responses
+include::partial$static_restapi/admin/index.adoc[tags=put_db-_user-name-responses,leveloffset=-3]
```

Comment:
As above, these snippets are wrong and they need to be https://docs.couchbase.com/sync-gateway/current/rest_api_admin.html#tag/Database-Security/operation/post_db-_user- (create) or https://docs.couchbase.com/sync-gateway/current/rest_api_admin.html#tag/Database-Security/operation/put_db-_user-name (update)